src/plugins/vis_type_xy/public/components/xy_settings.tsx

```diff
@@ -60,23 +60,15 @@ type XYSettingsProps = Pick<
   legendPosition: Position;
 };
 
-function getValueLabelsStyling(isHorizontal: boolean) {
-  const VALUE_LABELS_MAX_FONTSIZE = 15;
+function getValueLabelsStyling() {
+  const VALUE_LABELS_MAX_FONTSIZE = 12;
   const VALUE_LABELS_MIN_FONTSIZE = 10;
```

Comment:
I thought you were going to reduce the minimum, but I understand you wanted to have generally smaller font size. If the font size window is this small, is it worth just using fixed size as before?

Reply:
No strong opinion here, I prefer to have min max, even if the window is small.